### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by agarzon48** [Lucaso1992/Instagram-project-clase-1#1]
Muy bien, se han corregido las observaciones. Como extra podrías intentar hacerlo responsive, ya que en estos momentos la tarjeta siempre mide 600px de ancho. También convendría que no establecieras manualmente la altura: el div tomará automáticamente el espacio necesario para albergar todos los elementos

**Comment by Lucaso1992** [comment]
Hola Adrian, te hago una consulta. Para hacerlo responsive, estaria bien si el ancho lo establezco en rem y la altura directamente no la establezco? acabo de realizar estas modificaciones y creo que esta bien el resultado, full responsive. Pero queria confirmar nomas. Gracias, saludos.

**Comment by Lucaso1992** [comment]
es medio complejo verlo por aca, si quieres en la proxima mentoria lo vemos...

**Comment by agarzon48** [comment]
Sí, mentorías para resolver dudas, por aquí solo anotaciones! :)
